build_tools/benchmarks/shark/run_shark.sh

```diff
@@ -14,26 +14,34 @@
 #        --sha <SHA of https://github.com/nod-ai/SHARK.git to pin to> \
 #        --pytest-regex e.g. "cpu", "cuda", "cuda and torch". \
 #        --driver e.g. "cpu", "cuda", "vulkan" \
+#        --save-version-info <true|false> \
+#        --use-local-iree <true|false> \
+#        --iree-source-dir <source directory of local iree> \
 #        --output-dir <local output dir> \
-#        --save-version-info <true|false>
 
 set -xeuo pipefail
 
 export SAVE_VERSION_INFO=false
 export BENCHMARK_REGEX="cpu"
 export DRIVER="cpu"
+export USE_LOCAL_IREE=false
+export IREE_SOURCE_DIR="$(pwd)"
```

Comment:
Seems simpler to use the existing env variable rather than spawning a subshell here.

```suggestion
export IREE_SOURCE_DIR="${PWD}"
```

Reply:
Updated